sample.env

```diff
@@ -270,19 +275,31 @@ CM_DATABASE_STORAGE_COLLECTION_NAME=MongoStorage
 CM_DATABASE_SIZE_GB=50
 CM_DATABASE_SIZE_TARGET_PERCENT=0.8
 CM_DATABASE_DELETE_THRESHOLD_PERCENT=0.9
-CM_DATABASE_MAX_TTL_RETENTION_SECONDS=5184000 # 60 days
+CM_DATABASE_MAX_TTL_RETENTION_SECONDS=31536000 # 1 year
 CM_DATABASE_MIN_TTL_RETENTION_SECONDS=604800 # 7 days
 CM_DATABASE_COMPACTION_TRIGGER_PERCENT=0.5
 
 
 ##############################################################
-# jpo-conflictvisualizer cvmanager api environment variables #
+# cvmanager intersection api environment variables #
 # Documentation: https://github.com/usdot-jpo-ode/jpo-conflictvisualizer/tree/cvmgr-cimms-integration/api
 ##############################################################
 # Set the HOST IP of where the containers are running
 KAFKA_BROKER_PORT=9092
+KAFKA_BOOTSTRAP_SERVERS=${DOCKER_HOST_IP}:9092
+
+# GitHub Token (Required for Intersection API) - See services/intersection-api/README.md#github-token for steps to generate
+MAVEN_GITHUB_TOKEN=
+MAVEN_GITHUB_ORG=usdot-jpo-ode
+
+#Specify MongoDB connection parameters
+DB_HOST_IP=${DOCKER_HOST_IP}
+DB_HOST_PORT=27017
+CM_MONGO_READ_WRITE_USERNAME=api
+CM_MONGO_READ_WRITE_USERNAME=api

```

Comment:
You have duplicate CM_MONGO_READ_WRITE_USERNAME variables. I imagine one is meant to be a password? I also don't see this even getting used in the docker-compose files. Probably should just delete redundant variables if we have them.